test/uk/gov/hmrc/helptosavefrontend/controllers/RegisterSpec.scala

```diff
@@ -100,6 +104,11 @@ class RegisterSpec extends UnitSpec with WithFakeApplication with MockFactory {
       .expects(*,*,*)
       .returning(Future.successful(cacheMap))
 
+  def mockSessionCacheConnectorGet:Unit =
+    (mockSessionCacheConnector.get(_: HeaderCarrier,_:Reads[HTSSession]))
+      .expects(*,*)
```

Comment:
You can actually test the input to the session cache connector is what you expect here